tests/test_host/test_external_repo.py

```diff
@@ -9,20 +9,17 @@
 
 class ExternalRepoTest(unittest.TestCase):
     def test_download(self):
-        host_paths = paths.host()
+        tmp_subdir = pathlib.Path(paths.host().tmp_dir, "test-external-repo-subdir")
+        external_repo_dir = pathlib.Path(paths.get_external(tmp_subdir))
         repo = "twosixlabs/armory-example"
         repo_name = repo.split("/")[-1]
 
-        download_and_extract_repo(repo)
-        self.assertTrue(os.path.exists(f"{host_paths.external_repo_dir}/{repo_name}"))
-        self.assertTrue(
-            os.path.isfile(
-                str(
-                    pathlib.Path(
-                        f"{host_paths.external_repo_dir}/{repo_name}/README.md"
-                    )
-                )
-            )
-        )
+        download_and_extract_repo(repo, external_repo_dir=external_repo_dir)
+        basedir = external_repo_dir / repo_name
 
-        shutil.rmtree(str(pathlib.Path(f"{host_paths.external_repo_dir}/{repo_name}")))
+        self.assertTrue(os.path.exists(basedir))
+        self.assertTrue(os.path.isfile(basedir / "README.md"))
+        shutil.rmtree(basedir)
+        os.remove(external_repo_dir / (repo_name + ".tar.gz"))
+        os.rmdir(external_repo_dir)
+        os.rmdir(tmp_subdir)
```

Comment:
Hmmm wouldn't this work?
```suggestion
        shutil.rmtree(tmp_subdir)
```